JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -155,6 +155,10 @@
         public static dump(object: any): string {
             var objectTypeDump: string = Object.prototype.toString.call(object);
             var propertyValueDump: string = JSON.stringify(object);
+            if (objectTypeDump === "[object Error]") {
+                propertyValueDump = "{ stack: '" + object.stack + "', message: '" + object.message + "', name: '" + object.name + "'";
```

Comment:
I don't think `object.stack` will work for all browsers. Chrome's Error does not have that property on its prototype. I would much rather loop over properties and dump them into a new json.